test/copy/copy_npy_test.cpp

```diff
@@ -0,0 +1,313 @@
+#include <string>
```

Comment:
I think you don't need to include <string>

Reply:
Removed